web/controllers/password_reset_controller.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Looseleaf.PasswordResetController do
+  use Looseleaf.Web, :controller
+  alias Looseleaf.{PasswordResetter, Repo, User}
+
+  plug :put_layout, "home.html"
+
+  def new(conn, _params) do
+    conn
+    |> render("new.html")
+  end
+
+  def create(conn, %{"password_reset" => %{"email" => email}}) do
+    PasswordResetter.reset_password(email, Repo)
```

Comment:
Nit but this service call might fail and return an `{:error, changeset}` – should you handle?